### PR TITLE
Change autosubmit comment from 'You' to 'The PR author'

### PR DESCRIPTION
**Pull request by gmackall**
Example of the current bot comment:
>auto label is removed for flutter/flutter/138973, due to This PR has not met approval requirements for merging. You are not a member of flutter-hackers and need 1 more review(s) in order to merge this PR.
>
>Merge guidelines: You need at least one approved review if you are already part of flutter-hackers or two member reviews if you are not a flutter-hacker before re-applying the autosubmit label. Reviewers: If you left a comment approving, please use the "approve" review action instead.

This message is confusing when the person adding the "autosubmit" label is not the author of the PR (which is common in the case that the author is not a regular contributor), because the use of 'you' can be interpreted as 'the person who added the label'.

Context https://discord.com/channels/608014603317936148/608021351567065092/1179214124538921100

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: auto_submit/lib/validations/approval.dart [hunk 1]
```diff
@@ -57,19 +57,19 @@ class Approval extends Validation {
...
         approvedMessage =
             'This PR has not met approval requirements for merging. Changes were requested by ${approver.changeRequestAuthors}, please make the needed changes and resubmit this PR.\n'
-            '$flutterHackerMessage and need ${approver.remainingReviews} more review(s) in order to merge this PR.\n';
+            '$flutterHackerMessage and needs ${approver.remainingReviews} more review(s) in order to merge this PR.\n';
       } else {
         // No changes were requested so check approval count.
         approvedMessage = approved
             ? 'This PR has met approval requirements for merging.\n'
-            : 'This PR has not met approval requirements for merging. $flutterHackerMessage and need ${approver.remainingReviews} more review(s) in order to merge this PR.\n';
+            : 'This PR has not met approval requirements for merging. $flutterHackerMessage and needs ${approver.remainingReviews} more review(s) in order to merge this PR.\n';
```
Comment (Hixie) [inline auto_submit/lib/validations/approval.dart:72]: bonus points for making the `(s)` use the right plural in each case here and above. :-)

**Review by Hixie — APPROVED** [review]
LGTM

**Comment by gmackall** [comment]
@ricardoamador do you want to do any additional review as codeowner, or are you fine landing this with the current approvals?

**Review by ricardoamador — APPROVED** [review]
LGTM

**Pull request merged**